draft-ietf-dnsop-svcb-https.md

```diff
@@ -659,6 +661,36 @@ each RRSet in the Additional section with the same DNSSEC-related records
 that they would send when providing that RRSet as an Answer (e.g. RRSIG, NSEC,
 NSEC3).
 
+## EDNS Client Subnet (ECS) {#ecs}
+
+The EDNS Client Subnet option (ECS, {{!RFC7871}}) allows recursive
+resolvers to request IP addresses that are suitable for a particular client
+IP range.  IP addresses appear in two places in Authoritative SVCB responses:
+
+* ipv*hint SvcParams in a ServiceMode response.  Clients only use these hints
+when the recursive resolver does not provide A/AAAA records for TargetName in
+the Additional section, as recommended in {{recursive-behavior}}.
+* A/AAAA records returned by the Authoritative server in the Additional
+section of a SVCB response when TargetName is in-zone, as recommended in
+{{authoritative-behavior}}.
+
+SVCB records SHOULD NOT otherwise vary based on ECS.
+
+If the resolver plans to resolve A/AAAA records for TargetName and return
+them in the Additional section, as recommended in {{recursive-behavior}},
+it SHOULD include an ECS option with a SOURCE PREFIX-LENGTH of zero in SVCB
+queries, to indicate that ECS is supported but is disabled for this query.
+If the resolver does not plan to perform this resolution, it SHOULD send the
+same ECS option that it would use for an A/AAAA query.
+
+According to Section 7.3.1 of {{!RFC7871}}, "Any records from \[the
+Additional section\] MUST NOT be tied to a network".  Accordingly,
+resolvers SHOULD assume that any A/AAAA records in the Additional section
+are network-invariant.  When responding to a SVCB query with an ECS
+option (even one whose SOURCE PREFIX-LENGTH is zero), Authoritative servers
+must omit such records if they wish to serve subnet-specific addresses.
```

Comment:
```suggestion
MUST omit such records if they wish to serve subnet-specific addresses.
```

???

Reply:
"MUST if you wish" in all-caps seems strange to me.